server/controllers/admin.py

```diff
@@ -290,7 +302,7 @@ def autograde_backup(bid):
     return redirect(url_for('.grading', bid=bid))
 
 @admin.route("/versions/<name>", methods=['GET', 'POST'])
-@is_staff()
+@is_admin()
```

Comment:
We should make the argument to `is_staff` required (i.e. `is_staff(course_arg)` instead of `is_staff(course_arg=None)`) so that we're not tempted to use it, or make sure we got all the uses